src/enzyme_ad/jax/Implementations/HLODerivatives.td

```diff
@@ -1234,3 +1234,5 @@ def : HLODerivative<"TransposeOp", (Op $x),
                     ],
                     (SelectIfActive $x, (Transpose (ResultTypes), (Shadow $x), (Permutation)), (HLOConstantFP<"0">))
                   >;
+
+def : HLOInactiveOp<"BatchNormInferenceOp">;
```

Comment:
is this really inactive?

Reply:
yeah probably not a good idea. more like this being used inside AD is almost always a mistake